comp/core/flare/flare.go

```diff
@@ -163,7 +163,7 @@ func (f *flare) Create(pdata ProfileData, ipcError error) (string, error) {
 	if fb.IsLocal() {
 		// If we have a ipcError we failed to reach the agent process, else the user requested a local flare
 		// from the CLI.
-		msg := []byte("local flare was requested")
+		msg := []byte(fmt.Sprintf("local flare was requested\nFlare creation time: %s", time.Now().Format(time.RFC3339)))
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why only if it's local ? 
Why not just add a log line to `flare_creation.log` ?

Reply:
The card maxime created was just for the local mode, I think it may be due to the fact that we can find it in other file when not in local mode

Reply:
> you can add it all the time, but in non-local mode you get the status page with the date in it.